netsome/constants.py

```diff
@@ -33,15 +30,23 @@ class IPV4(enum.IntEnum):
 
 
 class VLAN(enum.IntEnum):
-
     VID_MIN = 0
     VID_MAX = 2**12 - 1
 
     VID_DEFAULT = 1
 
 
 class BGP(enum.IntEnum):
-
     ASN_MIN = 0
     ASN_MAX = BYTES.FOUR - 1
     ASN_ORDER_MAX = BYTES.TWO - 1
+
+
+PORT_PATTERNS = {
```

Comment:
might be usefull to use long and short port names/types, like Eth1/1 & Ethernet1/1